crates/noirc_evaluator/src/brillig/brillig_gen/brillig_slice_ops.rs

```diff
@@ -260,7 +260,7 @@ mod tests {
         arguments: Vec<BrilligParameter>,
         returns: Vec<BrilligParameter>,
     ) -> BrilligContext {
-        let mut context = BrilligContext::new(arguments, returns);
+        let mut context = BrilligContext::new(arguments, returns, true);
```

Comment:
Is `create_context` only for testing?

Reply:
Yes only for testing

Reply:
We should probably add `#[cfg(test)]` to it then or put it in a test module.

Reply:
Ok I'll do that

Reply:
It is actually within a tests module